Switch from python(2) to python3
--- a/Makefile
+++ b/Makefile
@@ -1,11 +1,11 @@
 # create a full source package
 sdist: doc
-	python setup.py sdist
+	python3 setup.py sdist
 	-# mv dist/pyang-*.tar.gz dist/pyang_src-*.tar.gz
 
 # create a minimal package
 dist: doc
-	python setup.py sdist
+	python3 setup.py sdist
 
 .PHONY:	test tags clean doc
 doc:
@@ -17,7 +17,7 @@
 clean:
 	(cd test && $(MAKE) clean)
 	(cd doc &&  $(MAKE) clean)
-	python setup.py clean --all
+	python3 setup.py clean --all
 	rm -rf build dist MANIFEST
 	find . -name "*.pyc" -exec rm {} \;
 
